moped-editor/src/queries/project.js

```diff
@@ -103,7 +103,7 @@ export const SUMMARY_QUERY = gql`
       entity_id
       entity_name
     }
-    moped_types {
+    moped_types(order_by: { type_name: asc }) {
```

Comment:
The only thing that was confusing is that we order by `type_name` when we also have a `type_order` column in the DB. In reading through, it seemed like @amenity called for that change. Just hope its not confusing why we have that `type_order` column in the future.

Reply:
wanna make a `Tech Debt` issue to clean this one up?